pkg/parse/root_test.go

```diff
@@ -118,56 +118,7 @@ func TestRoot_Parse(t *testing.T) {
 				fake.UnstructuredAtPath(kinds.Namespace(),
 					"",
 					core.Name("foo"),
-					core.Label(metadata.ManagedByKey, metadata.ManagedByValue),
-					core.Annotation(common.LifecycleDeleteAnnotation, common.PreventDeletion),
-					core.Annotation(metadata.ResourceManagementKey, metadata.ResourceManagementEnabled),
-					core.Annotation(metadata.GitContextKey, nilGitContext),
-					core.Annotation(metadata.SyncTokenAnnotationKey, ""),
-					core.Annotation(metadata.OwningInventoryKey, applier.InventoryID(rootSyncName, configmanagement.ControllerNamespace)),
-					core.Annotation(metadata.ResourceIDKey, "_namespace_foo"),
-					difftest.ManagedBy(declared.RootReconciler, rootSyncName),
-				),
-				fake.Role(core.Namespace("foo"),
-					core.Label(metadata.ManagedByKey, metadata.ManagedByValue),
-					core.Label(metadata.DeclaredVersionLabel, "v1"),
-					core.Annotation(metadata.DeclaredFieldsKey, `{"f:metadata":{"f:annotations":{},"f:labels":{}},"f:rules":{}}`),
-					core.Annotation(metadata.SourcePathAnnotationKey, "namespaces/foo/role.yaml"),
-					core.Annotation(metadata.ResourceManagementKey, metadata.ResourceManagementEnabled),
-					core.Annotation(metadata.GitContextKey, nilGitContext),
-					core.Annotation(metadata.SyncTokenAnnotationKey, ""),
-					core.Annotation(metadata.OwningInventoryKey, applier.InventoryID(rootSyncName, configmanagement.ControllerNamespace)),
-					core.Annotation(metadata.ResourceIDKey, "rbac.authorization.k8s.io_role_foo_default-name"),
-					difftest.ManagedBy(declared.RootReconciler, rootSyncName),
-				),
-			},
-		},
-		{
-			name:   "implicit namespace if unstructured, present and self-managed",
```

Comment:
This test case is no longer relevant; there will be no such thing as an "implicit and self-managed" namespace.